### DISCUSSION TITLE
Correlation between http request and response using OpenTelemery / Opentelemetry Collector

### DISCUSSION THREAD

**Discussion by neerajCL** [open-telemetry/community#1839] [Community Discussions]
How can we correlate http request and response using Opentelemetry / Opentelemetry Collector. 
What parameters we need to correlate.Can you suggest some good example or POC's that I can refer

**Comment by trask** [comment]
> Can you suggest some good example or POC's that I can refer

check out https://github.com/open-telemetry/opentelemetry-demo
